starters/basic-starter/pages/[...slug].tsx

```diff
@@ -43,11 +45,12 @@ export default function NodePage({ node, menus }: NodePageProps) {
 
 // Use the 'paths' key to specify wanted paths to be pre-rendered at build time.
 // See https://nextjs.org/docs/basic-features/data-fetching/get-static-paths.
-export async function getStaticPaths(): Promise<GetStaticPathsResult> {
+export async function getStaticPaths(context): Promise<GetStaticPathsResult> {
+  const limit = 200; // Change as desired.
```

Comment:
We should explain here _why_ we are limiting the maximum size of the static paths